app/controllers/concerns/course/statistics/live_feedback_concern.rb

```diff
@@ -6,10 +6,10 @@ def initialize_student_hash(students)
     students.to_h { |student| [student, nil] }
   end
 
-  def fetch_hash_for_live_feedback_assessment(submissions, live_feedback_codes, students)
+  def fetch_hash_for_live_feedback_assessment(submissions, assessment_live_feedbacks, students)
```

Comment:
put all the changes in this file on this commit https://github.com/Coursemology/coursemology2/pull/7497/commits/d7f986c54545a036cff8f6a821ac6b37754e16cb